app/Core/Database.php

```diff
@@ -29,7 +29,10 @@ public static function getConnectionData () {
 	 * @return bool database status flag
 	 */
 	public static function initialize () : bool {
-		self::$database = new \mysqli ('localhost:3306', 'root', 'glitch123');
+		$config_data = parse_ini_file ("../app/server.ini");
```

Comment:
Can you use single quotes; double quotes cause php to attempt to parse variables inside the string.

Reply:
fixed